### DISCUSSION TITLE
Requests made to Stripe

### DISCUSSION THREAD

**Discussion by LickABrick** [Dokploy/dokploy#662] [General]
Hi, I am (very happily) selfhosting my own Dokploy instance. I noticed my dokploy instance (frontend) is making requests to Stripe. My question is why? I can understand the cloud/managed version uses Stripe for payments but in the selfhosted version nothing uses Stripe.

Would it be possible to add a env variable to completely disable the Stripe integration?

**Comment by Siumauricio** [comment]
Hi, can you guide me in what screen is doing stripe requests? I supposed to don't make requests on the self hosted version so I need to see very well this!

**Comment by LickABrick** [comment]
Haven't checked every page, but when I go to what it looks like any page I get this request a few moments after page load: ![image](https://github.com/user-attachments/assets/05753dd9-1f95-4198-89b3-76eff0824422)

When I refresh the page on `https://example.com/dashboard/projects`
I get a whole bunch of Stripe requests:
![image](https://github.com/user-attachments/assets/7d5c0d71-3870-4be3-b64c-e3a69cb3d203)

**Reply by Siumauricio** [reply]
Thank you, I will take a look

**Comment by LickABrick** [comment]
Seems to be fixed in #779
